magma_access_gateway_post_install/agw_post_install.py

```diff
@@ -159,15 +159,15 @@ def check_cloud_check_in(self):
             any(
                 entry["MESSAGE"]
                 for entry in journal_reader
-                if self.GOT_CLOUD_HEARTBEAT_MSG in entry["MESSAGE"]
+                if self.GOT_HEARTBEAT_MSG in entry["MESSAGE"]
             )
             or any(  # noqa: W503
                 entry["MESSAGE"]
                 for entry in journal_reader
-                if self.CLOUD_CHECKIN_SUCCESSFUL_MSG in entry["MESSAGE"]
+                if self.ORC8R_CHECKIN_SUCCESSFUL_MSG in entry["MESSAGE"]
             )
         ):
-            raise AGWCloudCheckinError()
+            raise Orc8rConnectivityError()
         else:
             return True
```

Comment:
Why does it return true? Should we simply return? Do we ever use the bool value?

Reply:
I think it was just for testing purposes. 
I've removed `else` and corresponding test.